spec/mainadditions.html

```diff
@@ -271,6 +271,258 @@ <h1>AvailableNamedTimeZoneIdentifiers ( ): a List of Time Zone Identifier Record
         1. Return _result_.
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-systemtimezoneidentifier" oldids="sec-defaulttimezone" type="implementation-defined abstract operation">
+      <h1>SystemTimeZoneIdentifier ( ): a String</h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>
+          It returns a String representing the host environment's current time zone, which is either <del>a String representing a UTC offset for which IsTimeZoneOffsetString returns *true*, or a primary time zone identifier</del><ins>a primary time zone identifier or an offset time zone identifier</ins>.
+        </dd>
+      </dl>
+
+      <emu-alg>
+        1. If the implementation only supports the UTC time zone, return *"UTC"*.
+        1. [declared="s"] Let _systemTimeZoneString_ be the String representing the host environment's current time zone, either: a primary time zone identifier; or an offset time zone identifier<ins> String _s_ for which ParseText(StringToCodePoints(_s_), |NormalizedUTCOffset|) returns a Parse Node</ins>.
+        1. <del>If IsTimeZoneOffsetString(_systemTimeZoneString_) is *true*, return _systemTimeZoneString_.</del>
```

Comment:
This weird line was left over from 262 editors' comments to move the assertions into the previous line. But after assertions were removed, this line became unnecessary.